rust/build_assert.rs

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! Build-time assertion.
+//!
+//! In Rust it is very common to use assertions extensively and defensively.
```

Comment:
I am not sure this line is needed -- it may be understood like the C side does not.

Reply:
Oh, I mean that in Rust it's very common to use assertions extensively to ensure soundness. Will reword that.